static/js/whyis.js

```diff
@@ -905,56 +905,43 @@ function whyis() {
             },
         };
     }]);
-    
-    app.factory("getLabel", ["$http", '$q', function($http, $q) {
-        var promises = {}
-        function getLabel(uri) {
-            if (getLabel.labels[uri] === undefined && promises[uri] === undefined) {
-                if (!promises[uri]) {
-                    var localPart = uri.split("#").filter(function(d) {return d.length > 0});
-                    localPart = localPart[localPart.length-1];
-                    localPart = localPart.split("/").filter(function(d) {return d.length > 0});
-                    localPart = localPart[localPart.length-1];
-                    getLabel.labels[uri] = localPart;
-                    promises[uri] = $q.defer();
-                    $http.get(ROOT_URL+'about?uri='+encodeURI(uri)+"&view=label")
-                        .then(function(data, status, headers, config) {
```

Comment:
`$http.get` returns a promise that can only take a callback of 1 parameter, not 4.